styles.css

```diff
@@ -31,312 +31,685 @@
+    font-size: 20px;
+    margin-bottom: 60px;
+    padding: 0 12px;
+  }
+
+  .about-me-headline {
+    width: 100%;
+    height: 351px;
+    background-image: url("images/illustration-1-about-me-mobile.jpg"), url("images/illustration-2-about-me-mobile.jpg");
+    background-repeat: no-repeat;
+    background-position: top right, bottom left;
+  }
+
+  .about-me-headline p {
+    color: #42526e;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 40px;
+    padding: 0 35px;
+  }
+
+  .knowledges {
+    display: grid;
+    align-content: center;
+    grid-template-columns: 1fr;
+    grid-template-rows: 3fr;
+    width: 90%;
+    margin-top: 60px;
+  }
+
+  .image-container {
+    margin-top: 20px;
+  }
+
+  .about-me-card {
+    width: 315px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border: 1px solid #d0d9d4;
+    margin-bottom: 15px;
+    background: #ebf0ee;
+    justify-self: center;
+    border-radius: 10px;
+  }
+
+  .subcategory-items {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
+    margin-bottom: 100px;
+  }
+
+  .subcategory-item {
+    padding: 12px 12px;
+    font-family: 'Inter', sans-serif;
+    background: #fff;
+    margin-right: 5px;
+    color: #36b37e;
+    margin-bottom: 15px;
+    border-radius: 10px;
+  }
+
+  .frameworks-card {
+    max-width: 369px;
+    margin-left: 5%;
+    margin-right: 5px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border: 1px solid #d0d9d4;
+    margin-bottom: 15px;
+    background: #ebf0ee;
+  }
+
+  /* Contact Form */
+  .footer {
+    height: 785px;
+    background-image: url("images/illustration-contact-form.svg");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .contact-form-control {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 40px;
+  }
+
+  .user-name {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+  }
+
+  .user-email {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+  }
+
+  .user-msg {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 114px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+    padding-top: 10px;
+  }
+
+  .contact-for-contact-info {
+    margin-bottom: 40px;
+  }
+
+  .portfolio-owner-email {
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    height: 0;
+    color: #42526e;
+  }
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  .container {
+    padding-left: 20px;
+  }
```

Comment:
- [ ] Kindly remove this extra `padding-left`  In the **Desktop** version look :arrow_down: 

- Your website 
![222222222](https://user-images.githubusercontent.com/57562869/159999845-ae29a64c-edb0-4f20-bcf7-cffd4235aa41.png)


- Original design
![Screenshot from 2022-03-25 00-33-20](https://user-images.githubusercontent.com/26767383/159991458-12711f79-c355-44c0-8fe6-225352588a58.png)